content/2016/france/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "EclipseCon France 2016"
+date: 2016-06-07T08:00:00-04:00
```

Comment:
I think the date here needs to be 2016-07-06

Reply:
You mean put day before month? 🤔